libs/core/src/lib/carousel/carousel.component.scss

```diff
@@ -18,3 +18,9 @@ fd-carousel:focus {
 [dir=rtl] .fd-carousel__button.fd-button > [class*=sap-icon] {
     transform: rotate(0deg);
 }
+
+.fd-carousel__button.fd-button.is-focus,
+.fd-carousel__button.fd-button:focus {
+    top: 50%;
+    left: initial;
+}
```

Comment:
I think this CSS rule is causing the carousel button to move on focus. This is what I'm seeing when I keyboard navigate to the carousel button:
<img width="427" alt="Screen Shot 2020-12-10 at 8 06 44 AM" src="https://user-images.githubusercontent.com/48103491/101797466-f2678580-3abe-11eb-8535-f507a4d639ca.png">